evap/staff/tests/test_views.py

```diff
@@ -1682,3 +1682,43 @@ def test_degree_form(self):
         self.assertIn("Successfully", str(response))
 
         self.assertTrue(Degree.objects.filter(name_de="Test", name_en="Test").exists())
+
+
+class TestSemesterQuestionnaireAssignment(ViewTest):
```

Comment:
ViewTest has this additional test that checks whether calling the url does not crash. Could you add the url and test_users analogous to the other ViewTests?